cmd/csi-node-driver-registrar/main.go

```diff
@@ -18,13 +18,15 @@ package main
 
 import (
 	"context"
-	"flag"
+	goflag "flag"
```

Comment:
How about get rid of goflag and only use pflag?
I am wondering if something like this will just do the magic.
`flag "github.com/spf13/pflag"`

Reply:
So I did switch this to pflag, but we still need to add "flag" flagset to allow configuring klog flags

Reply:
Sorry I am super unfamiliar to this, what goflag do we need to add to pflag? I thought in your change you already define all the flag in pflag?

Reply:
So klog uses the default `flag.FlagSet` for defining the flags: https://github.com/kubernetes/klog/blob/master/klog.go#L420-L441. By doing `pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)`, we'll be importing all the flags defined for `flag` into `pflag`. 

If we don't add the `flag.FlagSet` to `pflag`, then setting verbosity and other klog flags will fail:
```
➜ ./bin/livenessprobe -v=2
unknown shorthand flag: 'v' in -v=2
```

Reply:
Thanks a lot! This looks good to me!

Reply:
If I remember correctly, switching from flag to pflag is a breaking change because the single-dash options that were valid are no longer accepted.